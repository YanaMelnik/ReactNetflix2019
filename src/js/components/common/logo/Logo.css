.logo {
  font-weight: 700;
  color: #f34c70;
  margin-left: 30px;
  line-height: 50px;
}
