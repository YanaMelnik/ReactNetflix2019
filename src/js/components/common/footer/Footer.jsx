import React from 'react';
import './Footer.css';
import Logo from '../logo/Logo.jsx';

const Footer = () => <footer className='footer'>
    <Logo/>
  </footer>;

export default Footer;
