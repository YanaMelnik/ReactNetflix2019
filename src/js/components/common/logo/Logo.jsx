import React from 'react';

import './Logo.css';

const Logo = () => <div
    className='logo wrapper'>
    netflixroulette
  </div>;

export default Logo;
