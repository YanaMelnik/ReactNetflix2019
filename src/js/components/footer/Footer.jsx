import React from 'react';
import Logo from '../common/logo/Logo.jsx';

const Footer = () => <footer>
    <Logo/>
  </footer>;

export default Footer;
