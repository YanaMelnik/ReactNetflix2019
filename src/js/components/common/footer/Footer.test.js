import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';
import Logo from '../logo/Logo';

describe('Footer', () => {
    let component;
    beforeEach(() => {
        component = shallow(<Footer/>);
    });
    it('should render component correctly in case when all props defined', () => {
        expect(component).toMatchSnapshot();
    });

    it('should contain logo', () => {
        expect(component.find(Logo)).toHaveLength(1);
    });
});
