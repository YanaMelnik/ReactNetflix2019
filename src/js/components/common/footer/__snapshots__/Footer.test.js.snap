// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render component correctly in case when all props defined 1`] = `
<footer
  className="footer"
>
  <Logo />
</footer>
`;
